docs/self-hosting/advanced/model-list.mdx

```diff
@@ -12,7 +12,7 @@ tags:
 
 # Model List
 
-LobeChat supports customizing the model list during deployment. You can use `+` to add a model, `-` to hide a model, and use `model name=display name<extension configuration>` to customize the display name of a model, separated by English commas. The basic syntax is as follows:
+LobeChat supports customizing the model list during deployment. This configuration is done in the environment for each [model provider](model-providers). You can use `+` to add a model, `-` to hide a model, and use `model name=display name<extension configuration>` to customize the display name of a model, separated by English commas. The basic syntax is as follows:
```

Comment:
just [model providers](/docs/self-hosting/environment-variables/model-provider)

Reply:
Done, this syntax is what is currently used on `docs/self-hosting/environment-variables/model-provider.mdx`, FYI.